From: Jon Bernard <jbernard@debian.org>
Date: Sun, 27 Mar 2011 20:48:58 -0400
Subject: Remove subsys from initscripts
To: libcg-devel@lists.sourceforge.net

Debian uses /var/lock for process lockfiles. The use of /var/log/subsys
is RedHat specific, I believe.
This is obviously not compatible with non debian-based distributions.
Ideally, we could settle on a change that works for all parties.
---
 scripts/init.d/cgconfig.in |    2 +-
 scripts/init.d/cgred.in    |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/init.d/cgconfig.in b/scripts/init.d/cgconfig.in
index 3459a42..59b85fa 100644
--- a/scripts/init.d/cgconfig.in
+++ b/scripts/init.d/cgconfig.in
@@ -34,7 +34,7 @@ prefix=@prefix@;exec_prefix=@exec_prefix@;sbindir=@sbindir@
 CGCONFIGPARSER_BIN=$sbindir/cgconfigparser
 CONFIG_FILE=/etc/cgconfig.conf
 servicename=cgconfig
-lockfile=/var/lock/subsys/$servicename
+lockfile=/var/lock/$servicename
 
 #
 # Source LSB routines
diff --git a/scripts/init.d/cgred.in b/scripts/init.d/cgred.in
index 3677f2c..98a602f 100644
--- a/scripts/init.d/cgred.in
+++ b/scripts/init.d/cgred.in
@@ -63,7 +63,7 @@ fi
 # For convenience
 processname=cgrulesengd
 servicename=cgred
-lockfile="/var/lock/subsys/$servicename"
+lockfile="/var/lock/$servicename"
 pidfile=/var/run/cgred.pid
 
 start()
-- 
